src/Coalesce.Web/gulpfile.js

```diff
@@ -41,132 +41,134 @@ function getFolders(dir) {
+        };
+
+        for (var destinationDir in packages) {
+            gulp.src(paths.npm + packages[destinationDir])
+                .pipe(gulp.dest(paths.lib + destinationDir));
+        }
+    });
+
+gulp.task("copy-files", ["copy-lib", "ts", "copy-js"]);
+
+gulp.task("copy-js",
+    function () {
+        gulp.src(paths.scripts + "*.js")
+            .pipe(gulp.dest(paths.js));
+    });
+
+gulp.task("js:watch",
+    function () {
+        gulp.watch(paths.scripts + "/*.js", ["copy-js"]);
+    });
+
+
+gulp.task("sass",
+    function () {
+        return gulp.src(paths.styles + "/*.scss")
+            .pipe(sass({
+                includePaths: [paths.npm + "bootstrap-sass/assets/stylesheets"]
+            }).on("error", sass.logError))
+            .pipe(gulp.dest(paths.css));
+    });
+
+gulp.task("sass:watch",
+    function () {
+        gulp.watch([paths.styles + "/*.scss"], ["sass"]);
+    });
+
+
+gulp.task("ts:local",
+    function () {
+        gulp.watch([paths.scripts + "/*.ts"], ["ts"]);
+    });;
+
+gulp.task('ts', function () {
+    // compile the root generated code into an app.js file
+    var rootAppJsProject = typescriptCompiler.createProject('tsconfig.json', { outFile: 'app.js' });
+    var rootApp = gulp.src([paths.scripts + '/Generated/{Ko,ko}*.ts', paths.scripts + '/Partials/*.ts', '!' + paths.scripts + '/*.d.ts'])
+        .pipe(sourcemaps.init())
+        .pipe(rootAppJsProject());
 
-gulp.task("copy-js", function () {
-    gulp.src(paths.scripts + "*.js")
+    rootApp.dts
         .pipe(gulp.dest(paths.js));
-});
-
-gulp.task('js:watch', function () {
-    gulp.watch(paths.scripts + '/*.js', ['copy-js']);
-});
-
-
-gulp.task("sass", function () {
-    //gulp.src([paths.styles + '/*.scss', 'Areas/*/Styles/*.scss'])
-    // get the files from the root
-    gulp.src(paths.styles + '/*.scss')
-    .pipe(sass().on('error', sass.logError))
-    .pipe(gulp.dest(paths.css));
-
-    //// get the files from the areas
-    //gulp.src('Areas/**/Styles/*.scss')
-    //.pipe(sass().on('error', sass.logError))
-    //    .pipe(flatten({ includeParents: 1 }))
-    //    .pipe(rename(function(path) {
-    //        var originalPath = path.dirname;
-    //        path.dirname += '/css';
-    //}))
-    //.pipe(gulp.dest(paths.wwwroot));
-});
-
-gulp.task('sass:watch', function () {
-    gulp.watch([paths.styles + '/*.scss'], ['sass']);
-    //gulp.watch([paths.styles + '/*.scss', 'Areas/**/Styles/*.scss'], ['sass']);
-});
 
+    rootApp.js
+        .pipe(sourcemaps.write('.'))
+        .pipe(gulp.dest(paths.js));
 
-gulp.task('ts:local', function () {
     // now compile the individual page files
     var individualFileTypescriptProject = typescriptCompiler.createProject('tsconfig.json');
-    var individualTsResult = gulp.src([paths.scripts + '/*.ts', '!' + paths.scripts + '/{coalesce,Ko,ko}*.ts'])
+    var individualTsResult = gulp.src([paths.scripts + '/*.ts', '!' + paths.scripts + '/{intellitect,Ko,ko}*.ts'])
```

Comment:
This should still be 'coalesce', not 'intellitect'. If this was copied from the starter project, then the starter project may be wrong too.